frontend/src/routes/Search.tsx

```diff
@@ -82,12 +113,22 @@ const query = graphql`
 
 type Props = {
     q: string;
+    f: string;
+    start: string;
+    end: string;
     outcome: SearchQuery$data["search"];
 };
 
-const SearchPage: React.FC<Props> = ({ q, outcome }) => {
+const SearchPage: React.FC<Props> = ({ q, f, start, end, outcome }) => {
     const { t } = useTranslation();
     const router = useRouter();
+    const defaultFilter = f as Filter;
```

Comment:
And what happens when a Strolch just edits the URL to `f=banana`? There needs to be a runtime check somewhere to make sure only valid filters are inside this variable. Not necessarily (but potentially) here.

Reply:
In case if invalid value it should be treated as if the query parameter doesn't exist. So exactly how you did it now.